### DISCUSSION TITLE
Correct way to change Tooltip delay?

### DISCUSSION THREAD

**Discussion by ReneeGA2020** [microsoft/microsoft-ui-xaml#8882] [Q&A]
In the TooltipService class, I found no property to change the init delay of tooltip.

I just want to make it so that my tooltips show up immediately. Currently I can only think of setting the IsOpen value to true for the TooltipService object with code every time the control trigger its PointerEnter event. But I'm not sure that it's the best practice.

**Comment by daltzctr** [comment]
+1
